atlasdb-config/src/main/java/com/palantir/atlasdb/http/AtlasDbHttpClients.java

```diff
@@ -41,7 +44,12 @@ private AtlasDbHttpClients() {
             String uri,
             Class<T> type,
             AuxiliaryRemotingParameters parameters) {
-        return ConjureJavaRuntimeTargetFactory.DEFAULT.createProxy(trustContext, uri, type, parameters).instance();
+        return Suppliers.memoizeWithExpiration(
```

Comment:
So I thought this was the first thing we tried and with 30 minutes expiration it was OOMing. Why is the 30 minutes fine on develop, but not fine here?

Reply:
Unclear, I'm chalking it down to me doing it wrong the first time and trying to get it to work with everything that exists already. I'm struggling to figure out an alternate hypothesis tbh.

Reply:
ok, so I would say we probably want to keep the 30 minute period here, in case your original PR just didn't do the wiring correctly. Then it's a representative test case

Reply:
cool

Reply:
well no build is working because certs 😠